backend/db/refresh.py

```diff
@@ -65,6 +73,12 @@ def refresh_db(
         print(f"Database refresh incomplete. An exception occurred.", file=sys.stderr)
         raise err
 
+    if check_db_status_var('new_request_while_refreshing'):
+        # possibly look at the timestamp for sanity check
+        delete_db_status_var('new_request_while_refreshing')
+        refresh_db(use_local_database=use_local_database)
```

Comment:
Noticed a bug here! Should be: `use_local_db`. Fixing now!
<img width="418" alt="Screen Shot 2023-07-20 at 6 12 34 PM" src="https://github.com/jhu-bids/TermHub/assets/13045020/921ece98-0baa-4726-8d82-83a1927c360a">

I think I'm going to largely replace `use_local_database` with `use_local_db` just because it is shorter.